app/controllers/bgeigie_imports_controller.rb

```diff
@@ -95,6 +95,15 @@ def destroy
     end
   end
 
+  def kml
+    bgeigie_import = BgeigieImport.find(params[:id])
+    ::Actions::BgeigieImports::Kml.new(
```

Comment:
Do you have more info on this `Actions` concept? I'm confused why you'd go this route over letting ActionView process the ERB for you.